src/main/java/org/jabref/gui/importer/ImportEntriesDialog.java

```diff
@@ -49,13 +50,16 @@
 import jakarta.inject.Inject;
 import org.controlsfx.control.CheckListView;
 
+@SuppressWarnings("checkstyle:RegexpMultiline")
```

Comment:
Why this?

Reply:
It changing automaticity. I didn't do anything on that.